java-components/cli/src/main/java/io/github/redhatappstudio/jvmbuild/cli/builds/BuildLogsCommand.java

```diff
@@ -39,10 +55,13 @@ public class BuildLogsCommand implements Runnable {
     @CommandLine.Option(names = "-n", description = "The build number", defaultValue = "-1")
     int buildNo;
 
+    @CommandLine.Option(names = "-l", description = "Use legacy retrieval")
+    boolean legacyRetrieval = false;
```

Comment:
I kept the legacy retrieval for now so I could compare the logs being retrieved from the two methods. I guess once we're ok I can remove it? (or maybe in a future iteration)

Reply:
I would leave both, but use legacy as a fallback (i.e. lookup the results entry, if it is not present attempt to read it from kube).

This was it will still work even if results is not installed on the cluster. At this stage I want results to be an optional dependency rather than a hard requirement.